tools/buildsys/src/manifest.rs

```diff
@@ -369,7 +419,7 @@ pub struct BuildPackage {
     pub external_files: Option<Vec<ExternalFile>>,
     pub package_name: Option<String>,
     pub releases_url: Option<String>,
-    pub source_groups: Option<Vec<PathBuf>>,
+    pub source_groups: Option<Vec<String>>,
```

Comment:
This should stay as `PathBuf` - better to use a stronger type when we know the value corresponds to a path.
```suggestion
    pub source_groups: Option<Vec<PathBuf>>,
```